CITATION.cff

```diff
@@ -0,0 +1,26 @@
+cff-version: 1.2.0
+message: "If you use this software, please cite it as below."
+  
+title: "US Arctic Observation Network Benefit Tool"
+doi: 10.5281/zenodo.8403760
+  
+type: "software"
+license: ["MIT"]
+  
+version: 0.5.0
+date-released: "2023-10-03"
```

Comment:
`version` and `date-released` need to be updated with every release, so we should do that automatically. Check out how QGreenland does it. You'll need to switch from `bump2version` (it's a dead project) to `bump-my-version` because it adds support for regex-based replacement that can handle the `date-released` field.

Here's our PR which migrated from `bump2version` -> `bump-my-version`. https://github.com/nsidc/qgreenland/pull/721